Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM eclipse-temurin:17.0.9_9-jdk-focal
+FROM --platform=linux/amd64 eclipse-temurin:17.0.9_9-jdk-focal
```

Comment:
Do we only want to support linux ... this is done for preventing MacOS to generate problems due to M1 / docker bridging to `qemu` (real problem still unknown esp. regarding Rock images blocking on MacOS/docker)